react/README.md

```diff
@@ -11,19 +11,21 @@ To run locally:
 - Clone this repo locally
 - Run the following
 
-```
+```terminal
 npm i
 npm run start
 ```
 
 ## Testing
 
-To add more participants to your call, you can run additional windows locally or take advantage of the React demo currently hosted at [https://partyline.daily.co/ ](https://partyline.daily.co/).
+To add more participants to your call, you can run additional windows locally or take advantage of the React demo currently hosted at [partyline.daily.co](https://partyline.daily.co/).
 
 ## Deployment
 
-Once you've deployed your own server, make sure to update the necessary URLs as noted in: 
+Once you've deployed your own server, make sure to update the necessary URLs as noted in:
 
 - react/src/CallProvider.jsx
-- react/public/ios-bridge.html
 
+## A note for Safari users
+
+Safari prevents the auto playing of audio by default. You can enable this in Preferences > Websites > Auto-Play
```

Comment:
I defer to you on whether or not it's also worth mentioning how we had to go back and "Unmute" multiple times.